deployments/warp_routes/tETH/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Turbo ETH",
+  "symbol": "tETH",
+  "description": "Warp Route tETH",
+  "image": "",
```

Comment:
does this need to point to the logo or no?

Reply:
this is a custom format I used for synthetic metadata based on an example somewhere, all fields are optional